_studio/shared/src/libmfx_allocator_vaapi.cpp

```diff
@@ -71,10 +71,10 @@ unsigned int ConvertMfxFourccToVAFormat(mfxU32 fourcc)
         return VA_FOURCC_YV12;
     case MFX_FOURCC_AYUV:
         return VA_FOURCC_AYUV;
-#if defined (MFX_ENABLE_FOURCC_RGB565)
+#if (MFX_VERSION >= 1028)
```

Comment:
No, dedicated define ```MFX_ENABLE_FOURCC_RGB565``` is much better since we have it

Reply:
Done